roles/swift-common/meta/main.yml

```diff
@@ -22,5 +22,11 @@ dependencies:
     when: openstack_install_method == 'package'
   - role: openstack-distro
     project_name: swift
-    project_packages: "{{ swift.distro.project_packages }}"
+    dependent_packages:
```

Comment:
again follow other roles, these needs to be project_packages, define in role defaults.